packages/vite-plugin-cloudflare/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Are the change to vite expected?

Reply:
Oops, sorry I missed that it was in draft state.
Took a look here as I was wondering why the change got reverted.